### ISSUE TITLE
Reduce dependence of infra tests on external servers

### ISSUE THREAD

**Issue by gsnedders** [w3c/web-platform-tests#9057]
In https://github.com/w3c/web-platform-tests/commit/b4b9ec466d8f719da9876c00afcf709529f1e4a6 I added a bunch of markers to a number of tests, including "remote_network".

While we might want integration tests to check our "install x" support works properly, we can test the majority of this code without relying on external servers.

**Comment by foolip** [comment]
Is this about replying on servers to download foodrivers and the like, not about the tests themselves pointing to external servers? How would one reduce the former meaningfully?

**Comment by gsnedders** [comment]
@foolip Yes. I was envisioning mocking out the download side of things and being able to test things in isolation from external servers.
